EncounterMe/EncounterMe/Services/MailService.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Net.Mail;
+using System.Text;
+using EncounterMe.Views;
+
+namespace EncounterMe.Services
+{
+    class MailService
+    {
+        public MailService()
+        {
+      
+        }
+
+        public void OnSuccessfulRegistration(object source, RegistationEventArgs args)
+        {
+            MailMessage mail = new MailMessage();
+            SmtpClient SmtpServer = new SmtpClient("smtp.gmail.com");
+
+            mail.From = new MailAddress("encounterMePSI@gmail.com");
```

Comment:
Ar čia šitą mail reikia sukurti?

Reply:
Taip, iš jo bus siunčiami visi laiškai vartotojams